src/ac_copilot_trainer/modules/realtime_coaching.lua

```
@@ -1,360 +1,351 @@
-    return nil
-  end
-
-  local result = coachingHints.buildRealTime(cornerLabel, lastFeats, bestFeats)
-  if not result then
-    return nil
-  end
-
-  hintShownThisLap[key] = true
+--- Place-holder viewmodel for the empty state. Returned when the app has zero
+--- reference data — keeps both tiles painted with sensible copy.
+local function placeholderView(carSp, curSpeed)
```

Luacheck (1.2.0)

[warning] unused argument 'carSp'

(W212)